third_party/terraform/website/docs/r/compute_instance.html.markdown

```diff
@@ -51,7 +56,9 @@ resource "google_compute_instance" "default" {
   metadata_startup_script = "echo hi > /test.txt"
 
   service_account {
-    scopes = ["userinfo-email", "compute-ro", "storage-ro"]
+    # Google now recommends custom service accounts that have cloud-platform scope and permissions granted via IAM Roles.
```

Comment:
I think this can be simplified to indicate the current state rather than a change.
```suggestion
    # Google recommends custom service accounts that have cloud-platform scope and permissions granted via IAM Roles.
```

Would it make sense / be possible to add a link to the section of the docs explaining this recommendation in more detail?

Reply:
I don't think so. I'm assuming terraform users know what custom service accounts.